Core/tests/Unit/Exception/ServiceExceptionTest.php

```diff
@@ -41,4 +42,194 @@ public function testDoesntHaveServiceException()
         $ex = new ServiceException('foo');
         $this->assertFalse($ex->hasServiceException());
     }
+
+    public function testGetErrorInfoMetadataWithGrpc()
+    {
+        if (!extension_loaded('grpc')) {
+            $this->markTestSkipped('Must have the grpc extension installed to run this test.');
```

Comment:
Curious if this needs to exist, it doesn't seem like `ApiException` has anything in it that is dependent upon an extension, it just parses data differently depending on the source. Same question goes for the rest based test below.

Reply:
Yeah, I had passed the `ApiException` obj, however I kinda treated it as I would a system test. Fixed it now.